.eslintrc.json

```diff
@@ -26,23 +26,50 @@
     "no-array-constructor": "error",
     "no-bitwise": "error",
     "no-caller": "error",
+    "no-catch-shadow": "error",
     "no-constant-condition": [ "error", { "checkLoops": false } ],
     "no-empty": [ "error", { "allowEmptyCatch": true } ],
     "no-eval": "error",
+    "no-extend-native": "error",
+    "no-extra-bind": "error",
+    "no-extra-label": "error",
+    "no-floating-decimal": "error",
     "no-implicit-coercion": [ "error", { "string": true, "boolean": false, "number": false } ],
+    "no-implicit-globals": "error",
     "no-implied-eval": "error",
+    "no-label-var": "error",
     "no-loop-func": "error",
     "no-multi-spaces": "error",
+    "no-multi-str": "error",
     "no-multiple-empty-lines": [ "error", { "max": 1, "maxBOF": 0, "maxEOF": 0 } ],
+    "no-native-reassign": "error",
+    "no-negated-in-lhs": "error",
     "no-new": "error",
     "no-new-func": "error",
     "no-new-object": "error",
+    "no-new-require": "error",
+    "no-new-wrappers": "error",
+    "no-octal-escape": "error",
+    "no-proto": "error",
+    "no-return-assign": "error",
     "no-script-url": "error",
+    "no-self-compare": "error",
+    "no-sequences": "error",
+    "no-shadow-restricted-names": "error",
     "no-spaced-func": "error",
+    "no-throw-literal": "error",
     "no-trailing-spaces": "error",
+    "no-undef-init": "error",
     "no-underscore-dangle": "error",
+    "no-unmodified-loop-condition": "error",
     "no-unneeded-ternary": [ "error", { "defaultAssignment": false } ],
+    "no-unused-expressions": "error",
+    "no-unused-vars": [ "error", { "args": "none" } ],
```

Comment:
`no-unused-vars` is part of eslint:recommended and already currently enforced without exception. I'm not sure we should loosen it to no longer check unused arguments.

<http://eslint.org/docs/rules/no-unused-vars>